frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/settings.ts

```diff
@@ -73,6 +73,6 @@ const handleCrowdedOrdinalXTicks = (
   }
 
   return xValuesCount > 10
-    ? assocIn(settings, ["x", "tick_display"], "rotate-45")
+    ? assocIn(settings, ["x", "tick_display"], "rotate-90")
```

Comment:
[In the new design](https://www.notion.so/metabase/Further-polish-static-viz-and-add-support-for-new-charts-29e3a1d2c55a4bf39dc0b1e5a0b2d67f#474ffaebb2c34b8790a511287f4781e7), we only rotate the label 90 degrees.